### PR TITLE
Consistent config toml files

### PR DESCRIPTION
**Pull request by ShibWizard**
# Description

Please provide a detailed description of what was done in this PR

# Changes

- [ ] Bugfix (non-breaking change that solves an issue)
- [ ] Hotfix (change that solves an urgent issue, and requires immediate attention)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)
- [ ] Changes only for a subset of nodes

# Breaking changes

Please complete this section if any breaking changes have been made, otherwise delete it

# Nodes audience

In case this PR includes changes that must be applied only to a subset of nodes, please specify how you handled it (e.g. by adding a flag with a default value...)

# Checklist

- [ ] I have added at least 2 reviewer or the whole pos-v1 team
- [ ] I have added sufficient documentation in code
- [ ] I will be resolving comments - if any - by pushing each fix in a separate commit and linking the commit hash in the comment reply
- [ ] Created a task in Jira and informed the team for implementation in Erigon client (if applicable)
- [ ] Includes RPC methods changes, and the Notion documentation has been updated

# Cross repository changes

- [ ] This PR requires changes to heimdall
    - In case link the PR here:
- [ ] This PR requires changes to matic-cli
    - In case link the PR here:

## Testing

- [ ] I have added unit tests
- [ ] I have added tests to CI
- [ ] I have tested this code manually on local environment
- [ ] I have tested this code manually on remote devnet using express-cli
- [ ] I have tested this code manually on mumbai
- [ ] I have created new e2e tests into express-cli

### Manual tests

Please complete this section with the steps you performed if you ran manual tests for this functionality, otherwise delete it

# Additional comments

Please post additional comments in this section if you have them, otherwise delete it

### CODE REVIEW

File: packaging/templates/puppynet/archive/config.toml [hunk 1]
```diff
@@ -90,10 +90,10 @@ gcmode = "archive"
     [jsonrpc.ws]
         enabled = true
         port = 8546
-        # prefix = ""
-        # host = "localhost"
-        # api = ["web3", "net"]
+        host = "0.0.0.0"
+        api = ["eth", "net", "web3", "txpool", "bor"]
```
Comment (vinayak0035) [inline packaging/templates/puppynet/archive/config.toml:94]: we are not adding debug ? not required ?

File: packaging/templates/puppynet/archive/config.toml [hunk 2]
```diff
@@ -119,7 +119,7 @@ gcmode = "archive"
     # maxheaderhistory = 1024
     # maxblockhistory = 1024
     # maxprice = "5000000000000"
-    # ignoreprice = "2"
+    # ignoreprice = "30000000000"
 
 [telemetry]
     metrics = true
```
Comment (vinayak0035) [inline packaging/templates/puppynet/archive/config.toml:125]: prometheus-addr = ""  if we are leaving it blank so after updating node we need to manually add 0.0.0.0:7071..

File: packaging/templates/puppynet/sentry/validator/bor/config.toml [hunk 3]
```diff
@@ -25,7 +25,7 @@ syncmode = "full"
     # debug = true
 
 [p2p]
-    maxpeers = 1
+    maxpeers = 20
```
Comment (vinayak0035) [inline packaging/templates/puppynet/sentry/validator/bor/config.toml:28]: here max peer will be 20 only.. can it have more peers than this.. like our own 9 validator.. 10 sentry if we extend 3 and seeds ?

File: packaging/templates/puppynet/without-sentry/bor/config.toml [hunk 4]
```diff
@@ -65,7 +65,7 @@ syncmode = "full"
 [miner]
     mine = true
     gaslimit = 20000000
-    # gasprice = "1000000000"
+    # gasprice = "30000000000"
```
Comment (vinayak0035) [inline packaging/templates/puppynet/without-sentry/bor/config.toml:68]: without sentry is same as validator only diff is max peers

File: packaging/templates/puppynet/archive/config.toml (hunk 1 above)
Comment (ShibWizard) [reply packaging/templates/puppynet/archive/config.toml:94]: No, We should not enable debug API by default

File: packaging/templates/puppynet/archive/config.toml (hunk 2 above)
Comment (ShibWizard) [reply packaging/templates/puppynet/archive/config.toml:125]: It was blank by default, If you think we are going to use in all nodes, Add in all 8 files, I have no issue.

File: packaging/templates/puppynet/sentry/validator/bor/config.toml (hunk 3 above)
Comment (ShibWizard) [reply packaging/templates/puppynet/sentry/validator/bor/config.toml:28]: This is for validator, 20 is anyway too much for val

File: packaging/templates/puppynet/archive/config.toml (hunk 2 above)
Comment (vinayak0035) [reply packaging/templates/puppynet/archive/config.toml:125]: yes we need it in all nodes .. adding

File: packaging/templates/puppynet/without-sentry/bor/config.toml (hunk 4 above)
Comment (ShibWizard) [reply packaging/templates/puppynet/without-sentry/bor/config.toml:68]: Yes, Without sentry means val without sentry so it should have more peers

File: packaging/templates/puppynet/archive/config.toml (hunk 1 above)
Comment (yannmart) [inline packaging/templates/puppynet/archive/config.toml:94]: debug flag

**Review by vinayak0035 — APPROVED** [review]

File: packaging/templates/puppynet/archive/config.toml (hunk 1 above)
Comment (ShibWizard) [reply packaging/templates/puppynet/archive/config.toml:94]: No, Should not be there by default, Generally RPCs doesn't provide debug apis

Comment (ShibWizard) [reply packaging/templates/puppynet/archive/config.toml:94]: If needed, We will add later

File: packaging/templates/shibarium/archive/config.toml [hunk 5]
```diff
@@ -80,7 +80,7 @@ gcmode = "archive"
     [jsonrpc.http]
         enabled = true
         port = 8545
-        host = "127.0.0.1"
+        host = "0.0.0.0"
         api = ["eth", "net", "web3", "txpool", "bor"]
```
Comment (yannmart) [inline packaging/templates/shibarium/archive/config.toml:84]: debug flag

Comment (ShibWizard) [reply packaging/templates/shibarium/archive/config.toml:84]: No, Not needed by default

**Review by yannmart — APPROVED** [review]

**Pull request merged**